src/js/share.js

```diff
@@ -15,7 +15,24 @@
 		email: 'mailto:?subject=See this article on FT.com&body={{title}}%0A{{url}}'
 	};
 
-	const shareUrlPromises = {};
+	const getShareUrl = memoize(function(serviceUrl, maxShares, context) {
+
+		maxShares = maxShares || 1;
+		context = context || 0;
+
+		return fetch(serviceUrl + '/generate' +
```

Comment:
As this function is not a part of the prototype, it does not have access to `config`, instead it takes `serviceUrl` as the first parameter.